content/news.pml

```diff
@@ -1159,293 +1159,8 @@
-   scenarios, including incremental cyber retrofit of legacy systems.
-
-   You can download <a href="https://sel4.systems/About/">here</a>.
-
-</p>
-</div>
-
-<!----------------------------------------------------------->
-<div class="news-item">
-  9 Jun 2020: Breakaway Consulting joins the seL4 Foundation!
-</div>
-<div>
- <p>
-
-   <a href="https://brkawy.com">
-   <img src="/Foundation/Membership/LOGOS/Brkawy.png"
-   style="width:40%; padding-left:1em; float:right"
-   alt="Breakaway logo">
-   </a>
-
-
-   We’re happy to announce the first new member since the launch of
-   the seL4 Foundation: Sydney-based company <a
-   href="https://brkawy.com">Breakaway Consulting</a>.
-
-   Through Founder and Managing Director Ben Leslie, Breakaway comes
-   with 20 years of experience in L4 microkernels and systems based on
-   various L4 kernels. Prior to founding Breakaway, Ben was a student
-   with what is now the TS Group, and then VP Engineering of Open
-   Kernel Labs.
-
-   Breakaway offers consulting services on architecture, design and
-   implementation of seL4-based systems.
-
-</p>
-</div>
-
-
-
-
-
-<!----------------------------------------------------------->
-
-<div class="news-item">
-  7 Apr 2020: Launch of the seL4 Foundation
-</div>
-
-<div>
-  <p>
-
-   <img src="/images/mosaic.jpg"
-   style="width:30%; padding-left:1em; float:right"
-   alt="seL4 foundation launch" />
-
-   The <a href="https://trustworthy.systems">Trustworthy Systems
-   group</a> is excited to announce the creation of the <a
-   href="/Foundation/">seL4 Foundation</a>. Its aim is to provide a
-   neutral and independent organisation to ensure the longevity of
-   seL4, and grow its ecosystem of adopters and contributors.
-
-   It has a <a href="/Foundation/Board/">high-profile Board</a>
-   consisting of Gernot Heiser (Chair), June Andronick, Gerwin Klein,
-   John Launchbury, Sascha Kegreiß, Daniel Potts.
-
-   It already has a number of members, including major adopters
-   HENSOLDT Cyber and Ghost Systems, and providers of services for
-   seL4, Cog Systems and DornerWorks. Please check the <a
-   href="/Foundation/Membership/">membership page</a> for details and
-   how to join.
-
-   The seL4 Foundation is set up under the Linux Foundation, which
-   provides a mature and well-known framework.
-  </p>
-
+<div class="news-finish">
   <p>
-
-   More information on the <a
-   href="https://microkerneldude.wordpress.com/2020/04/07/the-sel4-foundation-what-and-why/">Blog
-   post</a> about "The seL4 Foundation &mdash; What and Why", and in
-   the press releases from <a
-   href="https://www.linuxfoundation.org/press-release/2020/04/sel4-microkernel-optimized-for-security-gets-support-of-linux-foundation/">the
-   Linux Foundation</a> and from <a
-   href="https://www.csiro.au/en/News/News-releases/2020/seL4-developers-create-open-source-foundation">CSIRO's
-   Data61</a>.
+    Older news: <a href="news-20.pml">2020</a>
```

Comment:
Putting them to a separate page (and repeating this every year) will break the URLs that I try to add in the other PR for each news items. Now one could argue that if we do this in September each year most people wont be affected, but I would not rule out that eventually, older news item also will be referenced, and a permanent URL would be nice. At some point, there might be no way to handle this with just static web pages (without JavaScript and AJAX concepts) and a CMS becomes necessary.

Reply:
I agree that it's bad to change URLs, but that's not hard to achieve

Reply:
Simplest fix is to make a directory 'news', name the newsfiles 'news/20.pml', 'news/21.pml' etc; and make 'news.pml' just redirect to the latest.  That way, you never have to alter URLs.